pydantic/builder.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+"""
+
+Pydantic Models Builder
+=======================
+
+Build the pydantic models at the build of a registry by resolving the
+inheritance declaration and ForwardRefs type declaration into the models
+
+"""
+from typing import List, Optional
+
+import odoo
+from odoo import api, models as omodels
+
+from .registry import PydanticClassesRegistry, _pydantic_classes_databases
+
+
+class PydanticClassesBuilder(omodels.AbstractModel):
+    """Build the component classes
+
+    And register them in a global registry.
+
+    Every time an Odoo registry is built, the know pydantic models are cleared and
+    rebuilt as well.  The pydantic classes are built by taking every models with
+    a ``_name`` and applying pydantic models with an ``_inherits`` upon them.
```

Comment:
is _name still applicable to this version?

Reply:
no..